uniflow/op/model/lm/model_server.py

```diff
@@ -10,6 +10,8 @@
 from functools import partial
 from typing import Any, Dict, List, Optional
 
+import requests
```

Comment:
why are we using the requests instead of the AzureOpenAI client directly?

Reply:
just replaced requests with AzureOpenAI client